src/makeCalculator.js

```diff
@@ -5,6 +5,49 @@
  */
 function makeCalculator() {
   // write code here
+  return {
+    result: 0,
+    add: function (num) {
+      this.result += num;
+
+      return this;
+    },
+    subtract: function (num) {
+      this.result -= num;
+
+      return this;
+    },
+    multiply: function (num) {
+      this.result *= num;
+
+      return this;
+    },
+    divide: function (num) {
+      if (num !== 0) {
+        this.result /= num;
+
+        return this;
+      }
+    },
+    reset: function () {
+      this.result = 0;
+
+      return this;
+    },
+    operate(operation, num) {
```

Comment:
[Consider using this instead of multiple if statements](https://uk.javascript.info/call-apply-decorators)